src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/geoMap.js

```diff
@@ -84,14 +84,16 @@ const makeId = makeGenerator()
  * "longitude": -122.45,
  * "zoom": 15,
  * "controller": true,
+ * "isPickable": true,
```

Comment:
Please add a comment what this does. It sounds like the data point can be selected, but this PR is about tooltips. So, is this an additional feature, or is this required for a tooltip to show up?

Reply:
No, this is actually 1:1 with deck-gl isPickable field, for more info please check: https://deck.gl/docs/developer-guide/interactivity